roles/ipfs-cluster/templates/service.json

```diff
@@ -1,19 +1,66 @@
 {
+  "cluster": {
     "id": "{{ipfs_cluster_peer_id}}",
-    "private_key" : "{{ipfs_cluster_private_key}}",
-    "cluster_peers" : [
-    {% for host in groups['ipfs'] %}
-      "/ip4/{{ hostvars[host]['ansible_eth0']['ipv4']['address'] }}/tcp/9096/ipfs/{{ hostvars[host]['ipfs_cluster_peer_id'] }}"{% if not loop.last %},{% endif %}
-
-    {% endfor %}
+    "private_key": "{{ipfs_cluster_private_key}}",
+    "secret": "{{ipfs_cluster_secret}}",
+    "cluster_peers": [
+      {% for host in groups['ipfs'] %}
+      "/ip4/{{ hostvars[host]['ansible_default_ipv4']['address'] }}/tcp/9096/ipfs/{{ hostvars[host]['ipfs_cluster_peer_id'] }}"{% if not loop.last %},{% endif %}
+      {% endfor %}
```

Comment:
Can you double check this is not printing all multiaddresses on the same line? I think this removed the line-break between them and that's why there was an empty line before "endfor"

Reply:
Ah, I see, I'll put it back in!